RNAzureNotificationHub.podspec

```diff
@@ -0,0 +1,20 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "RNAzureNotificationHub"
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+
+  s.platform     = :ios, "7.0"
+  s.source       = { :git => "https://github.com/jonathanort/react-native-azurenotificationhub.git", :tag => "master" }
```

Comment:
Can you please update the source? I think it should be `https://github.com/CatalystCode/react-native-azurenotificationhub.git`.

Reply:
Thanks, this should now be resolved